core/keyserver/keyserver.go

```diff
@@ -102,8 +103,7 @@ func (s *Server) getEntry(ctx context.Context, userID string, epoch int64) (*tpb
 
 	neighbors, err := s.tree.NeighborsAt(ctx, index[:], epoch)
 	if err != nil {
-		log.Printf("Cannot get neighbors list: %v", err)
```

Comment:
In other words, all exported functions (which are indeed gRPC handlers) return gRPC errors. All unexported functions don't return gRPC errors.